### ISSUE TITLE
Resolve crt.sh <BR> 

### ISSUE THREAD

**Issue by m4ll0k** [aboul3la/Sublist3r#219]
- add this python code at `801` line

```python 
                if '<BR>' in subdomain:
                    sub = subdomain.split('<BR>')
                    for i in sub:
                        if i not in self.subdomain and i != self.domain:
                            self.subdomains.append(i)

```

**Comment by m4ll0k** [comment]
see #220

**Comment by aboul3la** [comment]
Hi,

Sorry for this late response.

I just pushed a fix for the issue in commit 61ebf366e096ffdc9fea6bb57f00683af5944bfc.
Please update your version to the latest version using "git pull" command.

Thank you!
